Essentials/src/main/java/com/earth2me/essentials/commands/Commandfly.java

```diff
@@ -37,9 +38,9 @@ protected void togglePlayer(final CommandSource sender, final User user, Boolean
                 user.getBase().setFlying(false);
             }
 
-            user.sendTl("flyMode", user.playerTl(enabled ? "enabled" : "disabled"), user.getDisplayName());
+            user.sendTl("flyMode", AdventureUtil.parsed(user.playerTl(enabled ? "enabled" : "disabled")), user.getDisplayName());
```

Comment:
I wonder if we could have a util methods for common placeholders like this, instead of pasting `AdventureUtil.parsed(user.playerTl(enabled ? "enabled" : "disabled"))` everywhere